test/unit/gemm/device/gemm_testbed_3x.hpp

```diff
@@ -1766,6 +1794,47 @@ bool TestAll(double alpha = 1.0, double beta = 0.0, CheckEquality check_relative
   return passed;
 }
 
+template <typename Gemm, template <class T> class ActivationFunctor =
+                             cutlass::epilogue::thread::Identity>
+bool TestXe(
+    double alpha = 1.0, double beta = 0.0,
+    CheckEquality check_relative_equality = CheckEquality::RELATIVE) {
+  using ElementScalar = typename Gemm::EpilogueOutputOp::ElementScalar;
+  using ProblemShapeType = typename Gemm::GemmKernel::ProblemShape;
+
+  Testbed3x<Gemm, ActivationFunctor> testbed(
+    check_relative_equality, ScalarLoc::ON_HOST, VectorBeta::DISABLED);
+
+  // For M & N we test a small and a big size
+  // For K, we currently only support K = TileShapeK
+  // We set L = 1 throughout
+  int max_alignment = 4; //std::max(Gemm::kAlignmentA, Gemm::kAlignmentB);
```

Comment:
TODO: the `Gemm::kAlignmentA` and `Gemm::kAlignmentB` members aren't defined for our `Gemm`. Hardcoded 4 here but should probably improve/generalise this?